dev-packages/e2e-tests/test-applications/tanstack-router/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "tanstack-router-e2e-test-application",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "build": "vite build",
+    "start": "vite preview",
+    "test": "playwright test",
+    "clean": "npx rimraf node_modules pnpm-lock.yaml",
+    "test:build": "pnpm install && npx playwright install && pnpm build",
+    "test:assert": "pnpm test"
+  },
+  "dependencies": {
+    "@sentry/react": "latest || *",
+    "@tanstack/react-router": "1.34.5",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "@types/react": "^18.2.66",
+    "@types/react-dom": "^18.2.22",
+    "@typescript-eslint/eslint-plugin": "^7.2.0",
+    "@typescript-eslint/parser": "^7.2.0",
+    "@vitejs/plugin-react-swc": "^3.5.0",
+    "typescript": "^5.2.2",
+    "vite": "^5.2.0",
+    "@playwright/test": "^1.41.1"
```

```suggestion
    "@playwright/test": "^1.41.1",
    "@sentry-internal/event-proxy-server": "link:../../../event-proxy-server"
```

I think this is missing here...?

---

It's odd but it doesn't seem to matter. I think because the package is available in the root node_modules.

---

yeah, I think that's why it works, we should still add it for safety/correctness I guess :D